be/src/exec/olap_scan_node.cpp

```diff
@@ -715,7 +714,7 @@ Status OlapScanNode::start_scan_thread(RuntimeState* state) {
 template<class T>
 Status OlapScanNode::normalize_predicate(ColumnValueRange<T>& range, SlotDescriptor* slot) {
     // 1. Normalize InPredicate, add to ColumnValueRange
-    RETURN_IF_ERROR(normalize_in_predicate(slot, &range));
+    RETURN_IF_ERROR(normalize_in_and_eq_predicate(slot, &range));
 
     // 2. Normalize BinaryPredicate , add to ColumnValueRange
     RETURN_IF_ERROR(normalize_binary_predicate(slot, &range));
```

Comment:
eq binary is also binary. So I think we would better also change the `normalize_binary_predicate` name.

Reply:
OK